test/word_test.cpp

```diff
@@ -0,0 +1,46 @@
+#include "doctest.h"
+
+#include "word.cpp"
+
+
+TEST_CASE("initialize member variables") {
+  Word w("gravy", "su");
+  CHECK(w.getVal() == "gravy");
+  CHECK(w.getFrequency() == 1);
+  CHECK(w.getStressPattern() == "su");
+
+}
+
+TEST_CASE("copy constructor") {
+  Word w("gravy", "su");
+
+  Word x = w;
+
+  CHECK(x.getVal() == "gravy");
+  CHECK(x.getFrequency() == 1);
+  CHECK(w.getStressPattern() == "su");
+
+  CHECK(&w != &x);
+}
+
+TEST_CASE("update frequency") {
+  Word w("gravy", "su");
+  w.incrementFrequency();
+
+  CHECK(w.getFrequency() == 2);
+}
+
+TEST_CASE("comparison operator") {
+  Word w("gravy", "su");
+  Word x("gravy", "su");
+
+  Word y("gravy", "uu");
+
+  Word z("notgravy", "su");
+  Word u("notgravy", "uu");
+
+  CHECK((w==x) == true);
+  CHECK((w==y) == true);
+  CHECK((w==z) == false);
+  CHECK((w==u) == false);
+}
```

Comment:
You shouldn't have to assert `== true` or `==false`. Can't you just do `CHECK(w==x);` and `CHECK(w!=z);`